config/config.go

```diff
@@ -29,6 +29,11 @@ import (
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
+const (
+	// minRangeMaxBytes is the minimum value for range max bytes.
+	minRangeMaxBytes = 1 << 20
```

Comment:
Any reason for this minimum? Might be useful to set `RangeMaxBytes` even lower in order to stress splitting and merging.